frontend/www/js/omegaup/components/course/AssignmentList.vue

```diff
@@ -149,17 +149,17 @@ export default class CourseAssignmentList extends Vue {
   contentOrderChanged = false;
   T = T;
   AssignmentFormMode = omegaup.AssignmentFormMode;
-  private _content: types.CourseAssignment[] = this.content;
+  course_AssignmentList_content: types.CourseAssignment[] = this.content;
```

Comment:
nit:

```suggestion
  $_course_AssignmentList_content: types.CourseAssignment[] = this.content;
```

Reply:
Justamente esa es la forma de variables que no me permite acceder. Intenté con el prefix `$...` y con `$_...`, pero con ninguno de los dos tuve éxito

Reply:
ah ya veo, me había quedado con la idea que el problema era _sólo_ el `private`. en ese caso,

```suggestion
  _content: types.CourseAssignment[] = this.content;
```

funciona?

Reply:
Parece que a esas variables sólo se puede acceder por medio de `$data`:
https://forum.vuejs.org/t/underscore-prefixes-for-vue-data/4018/3

Reply:
como ninguno de los métodos recomendados por la guía de estilo de Vue funcionan, dejémoslo como 

```suggestion
  currentContent: types.CourseAssignment[] = this.content;
```

Reply:
Listo